### DISCUSSION TITLE
Standup Section

### DISCUSSION THREAD

**Discussion by Prathamesh-Shanbhag** [EddieHubCommunity/support#869] [Ideas]
A Space here where people can add their daily standup and/or link their blog sites, and simultaneously get more familiar with using Github?

**Comment by aniketsingh98571** [comment]
Sure, Here you go -->https://techno-dict.hashnode.dev/my-experience-in-hacker-community

**Comment by eddiejaoude** [comment]
We have been talking about this in Discord. We currently have a channel for it, but having it on GitHub I think would be better 👍
